src/main/kotlin/no/nav/familie/dokument/storage/StorageController.kt

```diff
@@ -101,16 +101,24 @@ class StorageController(
             .body(mapOf("dokumentId" to uuid))
     }
 
-    // / TODO: "bucket"-path brukes ikke ennå. "familievedlegg" brukes alltid
     @GetMapping(path = ["{bucket}/{dokumentId}"], produces = [MediaType.APPLICATION_JSON_VALUE])
-    fun getAttachment(
+    fun getAttachmentRessurs(
         @PathVariable("bucket") bucket: String,
         @PathVariable("dokumentId") dokumentId: String,
     ): ResponseEntity<Ressurs<ByteArray>> {
+        return ResponseEntity.ok(Ressurs.success(getAttachment(bucket, dokumentId)))
+    }
+
+    // / TODO: "bucket"-path brukes ikke ennå. "familievedlegg" brukes alltid
+    @GetMapping(path = ["{bucket}/{dokumentId}"], produces = [MediaType.APPLICATION_OCTET_STREAM_VALUE])
+    fun getAttachment(
```

Comment:
Kan det blir noe "kluss" med de som bruker endepunktet her fra før? Hva blir "default" oppførsel når du ikke spesifiserer noe i kallet - ettersom det er samme endepunktene? 🤔

Reply:
Burde pathen spesifisere `/pdf` for å bemerke at det er dét man får ut? 🤔

Reply:
Kan vel sikkert bli det, endret til `/pdf` 👍